dev/PushNotifications/PushNotificationChannel.cpp

```diff
@@ -48,18 +51,75 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
 
     winrt::event_token PushNotificationChannel::PushReceived(winrt::Windows::TypedEventHandler<winrt::Microsoft::Windows::PushNotifications::PushNotificationChannel, winrt::Microsoft::Windows::PushNotifications::PushNotificationReceivedEventArgs> handler)
     {
-        return m_channel.PushNotificationReceived([weak_self = get_weak(), handler](auto&&, auto&& args)
+        if (IsPackagedApp())
```

Comment:
This check is incorrect. Packaged apps that don't use the BI APIs (unsupported) will need to subscribe their FG sink with the LRP. This is because, we will treat these apps as unpackaged essentially. I believe you need to use IsActivatorSupported here.

Reply:
This function checks if the app is Packaged/BiAvailable but the function name is not accurate. Will rename.